addon/classes/Column.js

```diff
@@ -333,7 +342,7 @@ export default class Column extends Ember.Object.extend({
 
     let { subColumns } = options;
 
-    subColumns = emberArray(makeArray(subColumns).map((sc) => new Column(sc)));
+    subColumns = emberArray(makeArray(subColumns).map((sc) => merge(sc, { parent: this })).map((sc) => new Column(sc)));
     subColumns.setEach('_group', this);
```

Comment:
I would vote for making it public.
what would we want to call it:
`parent`?
`parentColumn`?
`group`?